shared/native/notification-listeners.desktop.js

```diff
@@ -32,6 +33,10 @@ export default function (dispatch: Dispatch, getState: () => Object, notify: any
       dispatch(bootstrap())
       response.result()
     },
+    'keybase.1.NotifySession.clientOutOfDate': ({upgradeTo, upgradeURI, upgradeMsg}) => {
+      const body = upgradeMsg || `Please update to ${upgradeTo} by going to ${upgradeURI}`
+      NotifyPopup('Client out of date!', {body})
```

Comment:
Can you use the `notify` callback here? It looks like it's just equal to `NotifyPopup`: https://github.com/keybase/client/blob/429465b422a890105fb8b2271802eabea7d997d8/shared/actions/notifications.js#L31